erpnext_com/www/docs/user/manual/en/setting-up/chart-of-accounts-importer.md

```diff
@@ -0,0 +1,42 @@
+<!--add breadcrumbs-->
+
+# Chart Of Accounts Importer
+
+>Introduced in Version 12
+
+To start importing charts of accounts navigate to
+
+> Getting Started > Data Import and Settings > Chart of Accounts Importer
+
+1\. Select the Company for which you want to import chart of accounts. Make sure the company you are selecting doesn't have an pre existing transactions otherwise you'll receive a validation error.
```

Comment:
remove escape characters and use tabs to indent images and continue bullets

have 'a' pre existing NOT an